extensions/cornerstone/src/services/CornerstoneCacheService/CornerstoneCacheService.ts

```diff
@@ -83,14 +88,30 @@ class CornerstoneCacheService {
     return viewportData;
   }
 
+  /** This is called to compare the data source values */
+  public async invalidateStackViewport(
```

Comment:
You should probably use the `invalidateViewportData` that we have down here, and just branch for stack viewport